deploy/helm/sumologic/values.yaml

```diff
@@ -337,6 +337,15 @@ metrics-server:
 ## Configure fluent-bit 
 ## ref: https://github.com/helm/charts/blob/master/stable/fluent-bit/values.yaml
 fluent-bit:
+  ## Resource limits for fluent-bit
+  resources: {}
```

Comment:
I believe prometheus operator does not have default resource limits defined, can we confirm whether fluent-bit has any default resource limits? otherwise this would be overriding the defined defaults

Reply:
There are no default limits for 2.8.1 version: https://github.com/helm/charts/blob/866c1da688dc13d176a9b251ff4636a162c6a735/stable/fluent-bit/values.yaml#L187